crates/pera-core/src/dwallet_mpc/network_dkg.rs

```diff
@@ -253,3 +263,27 @@ fn generate_ristretto_dkg_party_public_input(
 ) -> DwalletMPCResult<Vec<u8>> {
     <DKGFirstParty as crate::dwallet_mpc::dkg::DKGFirstPartyPublicInputGenerator>::generate_public_input()
 }
+
+pub(crate) fn new_from_dkg_public_output(
```

Comment:
rename this func, it's not part of a Struct, hard to understand what object is created without reading the func.